assets/js/mosaic-setup.js

```diff
@@ -144,6 +146,26 @@ function customFunctionOnPgnGameLoad() {
     if (IsRotated && getDisplayedMinigame() != displayedGame) {
         flipMiniboard();
     }
+
+    // start clock countdown if enabled
```

Comment:
Please start all comments with uppercase letter (similar to the rest of the code)